StringCalculator_Kata/StringCalculator.cs

```diff
@@ -6,7 +6,7 @@ public class StringCalculator
     {
         public int Add(String numbers)
         {
-            String[] separator = { ",", "\n" };
+            String[] separator = { ",", "\n", "//", ":", "." ,";"};
```

Comment:
Why we are adding "//" , ":" , "." and ";" as delimiters ? 
I think there is misunderstanding of what how you can use different delimiters 
so if you want your delimiter to be # 
then you pass this string "//#\n1#2#4"